src/openfermion/resource_estimates/pbc/thc/utils/gvec_logic.py

```diff
@@ -0,0 +1,184 @@
+# coverage: ignore
```

Comment:
is this for the whole file? can we make it more specific? presumably some of these functions are tested in the tests

Reply:
Removed

Reply:
Eh actually, deleting this causes the workflow to fail, codecov complains the entire file is untested. Not sure why this is the case at the moment, given that the unit test tests the majority.